jest.config.js

```diff
@@ -1,9 +1,10 @@
 module.exports = {
-  preset: 'ts-jest',
+  preset: 'ts-jest/presets/default-esm',
```

Comment:
ESM support in conjunction with ts source, ref: https://kulshekhar.github.io/ts-jest/docs/guides/esm-support/